backend/app/routes/authorizationRoutes.ts

```diff
@@ -1,16 +1,16 @@
 import * as express from 'express'
 import RouterBase from '../interfaces/RouterBase'
-import AuthorizationController from '../controllers/authorizationController'
+import AuthController from '../controllers/authController'
 
 class AuthorizationRoutes implements RouterBase {
     public login = '/login'
     public loginVerify = '/login-verify'
     public logout = '/logout'
     public router = express.Router()
-    private authController: AuthorizationController
+    private authController: AuthController
 
     constructor() {
-      this.authController = new AuthorizationController()
+      this.authController = new AuthController()
```

Comment:
```suggestion
      this.authController = new AuthController()
      
```

Newline under var assignment. Next line is a func call.